src/settings-ui/Settings.UI/SettingsXAML/Controls/KeyVisual/KeyVisual.xaml

```diff
@@ -3,64 +3,56 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:controls="using:Microsoft.PowerToys.Settings.UI.Controls">
 
-    <x:Double x:Key="DefaultIconSize">16</x:Double>
-    <x:Double x:Key="SmallIconSize">12</x:Double>
-    <Style x:Key="DefaultTextKeyVisualStyle" TargetType="controls:KeyVisual">
-        <Setter Property="MinWidth" Value="56" />
-        <Setter Property="MinHeight" Value="48" />
-        <Setter Property="Background" Value="{ThemeResource AccentButtonBackground}" />
-        <Setter Property="Foreground" Value="{ThemeResource AccentButtonForeground}" />
-        <Setter Property="BorderBrush" Value="{ThemeResource AccentButtonBorderBrush}" />
-        <Setter Property="BorderThickness" Value="{ThemeResource ButtonBorderThemeThickness}" />
+    <Style BasedOn="{StaticResource DefaultKeyVisualStyle}" TargetType="controls:KeyVisual" />
+
+    <Style x:Key="DefaultKeyVisualStyle" TargetType="controls:KeyVisual">
```

Comment:
- Is this style based on its own? Or am I mistaken? 🤔

Reply:
Yup, this will automatically apply `DefaultKeyVisualStyle` to all controls that are of type `controls:KeyVisual` by the first style. We add the `x:Key="DefaultKeyVisualStyle"` so that these styles can be referenced (e.g. if you want to extend it). Common practice, e.g.: https://github.com/microsoft/microsoft-ui-xaml/blob/winui3/release/1.5-stable/controls/dev/Expander/Expander.xaml

Reply:
Okay. And can these two lines be merged into one, maybe?

Reply:
Nope, because whenever you set an `x:Key` is doesn't get applied automatically on every control of its target type.